src/Bundle/Resources/config/services/integrations/translation.xml

```diff
@@ -16,10 +16,8 @@
         <defaults public="true" />
 
         <service id="sylius.translation_locale_provider.immutable" class="Sylius\Resource\Translation\Provider\ImmutableTranslationLocaleProvider">
-            <argument type="collection">
-                <argument>%locale%</argument>
-            </argument>
-            <argument>%locale%</argument>
+            <argument>%kernel.enabled_locales%</argument>
```

Comment:
I'm not sure if this change requires targeting the PR to the next branch in this case?